part-spec/definitions.json

```diff
@@ -1,250 +1,339 @@
+        },
+        "vihMin": {
+          "description": "minimum voltage at which an input is read as high",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "vilMax": {
+          "description": "maximum voltage at which an input is read as low",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "vol": {
+          "description": "voltage ouput low or minimum voltage of a high output",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "voh": {
+          "description": "voltage ouput high or maximum voltage of a high output",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "abs_vmax": {
+          "description": "absolute maximum voltage that can be applied to a pin",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "abs_vmin": {
+          "description": "absolute minimum voltage that can be applied to a pin",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "vmax": {
+          "description": "maximum continuous voltage that can safely be applied to a pin",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "imax": {
+          "description": "maximum continuous current that can safely be drawn from a pin",
+          "comment": "units of amps",
+          "$ref": "#/definitions/unit"
+        },
+        "inputLeakage": {
+          "description": "maximum current draw into a high impedance input pin",
+          "comment": "units of amps",
+          "$ref": "#/definitions/unit"
+        },
+        "outputLeakage": {
+          "description": "maximum current flow from a pin during the off state",
+          "comment": "units of amps",
+          "$ref": "#/definitions/unit"
+        },
+        "dcResistance": {
+          "description": "resistance of a pin of a connector",
+          "comment": "units of ohms",
+          "$ref": "#/definitions/unit"
+        },
+        "voltageOptions": {
+          "description": "list of voltage levels supported by a pin",
+          "comment": "units of volts",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/unit"
+          }
+        },
+        "floatUnused": {
+          "description": "description of whether pin can safely be floated if it is not used",
+          "type": "boolean"
+        },
+        "internalPullUp": {
+          "description": "indicates the value of an internal pull-up on a pin",
+          "comment": "units of resistance",
+          "$ref": "#/definitions/unit"
+        },
+        "internalPullDown": {
+          "description": "indicates the value of an internal pull-down on a pin",
+          "comment": "units of resistance",
+          "$ref": "#/definitions/unit"
```

Comment:
I can no longer find the Unit definition section in this updated version. Was it deleted?